core/src/main/java/com/linecorp/armeria/common/multipart/Multipart.java

```diff
@@ -157,6 +176,32 @@ static Multipart from(String boundary, Publisher<? extends HttpData> contents, B
         return of(boundary, decoder);
     }
 
+    /**
+     * Checks whether provided {@link MediaType} is a multipart type.
+     * @param contentType {@link MediaType} to check against or {@code null} if undefined.
+     * @return true, only if provided {@link MediaType} is a multipart type.
+     */
+    static boolean isMultipart(@Nullable MediaType contentType) {
+        return contentType != null && MediaType.ANY_MULTIPART_TYPE.type().equals(contentType.type());
+    }
+
+    /**
+     * Extracts {@code boundary} parameter value from the multipart {@link MediaType}.
+     * @param contentType {@link MediaType} that represents on of the multipart subtypes.
+     * @return {@code boundary} parameter value extracted from the multipart {@link MediaType}.
+     */
+    static String getBoundary(MediaType contentType) {
```

Comment:
How about moving this method and `isMultipart()` to a utility class? I'm somewhat concerned that users may confuse `getBoundary()` and `boundary()`. We could add a new utility class `Multiparts` in the same package for example.

Reply:
I was expecting this comment and wanted to know what would you recommend. Sure, will move it to `Multiparts` then.

Reply:
Thanks, @max904-github !